run_scans.py

```diff
@@ -132,7 +132,8 @@ def launchTestsArgs(tool, shelf, slot, link, chamber, vfatmask, scanmin, scanmax
       pass
     log = file("%s/scanLog.log"%(dirPath),"w")
     if preCmd and config:
-      runCommand(preCmd,log)
+      #runCommand(preCmd,log)
+      runCommand(preCmd)
```

Comment:
Any reason why you removed the logging (or just commented it out)?

Reply:
?

Reply:
Removing the logging, is not the correct thing to do here.  If there's a problem, the problem should be addressed.
Logging in this manner has been working for a long time

Reply:
Still needs resolution

Reply:
NVM, see update now